_account-security/security/encryption/encryption-general.md

```diff
@@ -0,0 +1,113 @@
+  - [{{ section.title }}](#{{ section.anchor }})
+  {% endfor %}
+
+# -------------------------- #
+#           Content          #
+# -------------------------- #
+
+
+ssh-tunnels:
+  - name: "Self-hosted"
+    guide: "ssh-generic"
+    description: "If your database is hosted on your server and not in the cloud, it's considered a 'self-hosted' database. This is applicable to both integrations and destinations."
+
+  - name: "Amazon"
+    guide: "ssh-amazon"
+    description: "Stitch currently supports connecting Amazon RDS and Amazon Redshift (destination only) databases."
+
+  - name: "Microsoft Azure"
+    guide: "ssh-microsoft-azure"
+    description: "Stitch currently supports connecting Microsoft Azure SQL Server (as an integration) and Azure SQL Data Warehouse (as a destination). Other Microsoft Azure offerings aren't currently supported."
+
+sections:
+  - title: "SSL connections"
+    anchor: "ssl-connections"
+    content: |
+      [Secure Sockets Layer](https://www.verisign.com/en_US/website-presence/online/ssl-certificates/index.xhtml){:target="new"}, or SSL, is a standard security technology used to establish encrypted communication between a web server and a browser. SSL ensures that communication to and from Stitch remains private and secure.
+
+    subsections:
+      - title: "Stitch application access"
+        anchor: "stitch-application"
+        content: |
+          Access to the Stitch application enforces SSL to ensure your login credentials remain secure.
+
+      - title: "Connections that use verified SSL by default"
+        anchor: "connections-ssl-default"
+        content: |
+          For any connection using an HTTP API - for example, integrations like [Salesforce]({{ site.baseurl }}/integrations/saas/salesforce) or [Facebook Ads]({{ site.baseurl }}/integrations/saas/facebook-ads) - or Stitch's [Import API]({{ link.integrations.import-api | prepend: site.baseurl }}), Stitch will use [SSL/TLS-based encryption](https://www.verisign.com/en_US/website-presence/online/ssl-certificates/index.xhtml){:target="new"} by default.
+
+          This is also applicable to Stitch's [Amazon Redshift]({{ link.destinations.overviews.redshift | prepend: site.baseurl }}), [Google BigQuery]({{ link.destinations.overviews.bigquery | prepend: site.baseurl }}), [Microsoft Azure SQL Data Warehouse]({{ link.destinations.overviews.azure | prepend: site.baseurl }}), and [Snowflake]({{ link.destinations.overviews.snowflake | prepend: site.baseurl }}) destination offerings.
+
+          Connections to these integrations and destinations will attempt to use verified SSL with no action required on your part.
+
+      - title: "Connections with configurable SSL options"
+        anchor: "connections-configurable-ssl"
+        content: |
+          For non-web-hosted database connections, Stitch may support configurable SSL. To use SSL with a database Stitch supports, the database support and allow SSL connections.
```

Comment:
"non-web-hosted databases" is a confusing phrase. I think it's a little hard to draw a categorical line around the databases that we handle SSL for automatically versus those that we offer manual configuration for - I recommend just saying,  "For some databases..."

Also - missing a word or two at the end of this sentence, "the database support and allow..." should be something like, "the database must be configured to support and allow..."